core/src/org/pentaho/di/core/database/DatabaseInterface.java

```diff
@@ -1170,4 +1170,10 @@ public ValueMetaInterface customizeValueFromSQLType( ValueMetaInterface v, java.
    * @return String the create table statement
    */
   public String getCreateTableStatement();
+
+   /**
+   * Set default options for this database
+   */
+  public void addDefaultOptions();
```

Comment:
I don't believe a method can be added to this Interface in a non-major release. That is the purpose of DatabaseInterfaceExtended ... please see http://wiki.pentaho.com/display/PEOpen/API+Guidelines